adapter/internal/oasparser/model/adapter_internal_api.go

```diff
@@ -148,8 +148,9 @@ type EndpointConfig struct {
 
 // RetryConfig holds the parameters for retries done by apk to the EndpointCluster
 type RetryConfig struct {
-	Count       int32    `mapstructure:"count"`
-	StatusCodes []uint32 `mapstructure:"statusCodes"`
+	Count                int32    `mapstructure:"count"`
```

Comment:
add descriptions with default value info

Reply:
Added the default values in bakckend_types https://github.com/wso2/apk/pull/1324/files#diff-08af2ed2c7ea18e83d032f859784d0fc14168cab430ebbfe8b8571e2ad05d594R71